pkg/kubelet/cm/devicemanager/topology_hints_test.go

```diff
@@ -102,7 +102,7 @@ func TestGetTopologyHints(t *testing.T) {
 				return tc.expectedHints[r][i].LessThan(tc.expectedHints[r][j])
 			})
 			if !reflect.DeepEqual(hints[r], tc.expectedHints[r]) {
-				t.Errorf("%v: Expected result to be %v, got %v", tc.description, tc.expectedHints[r], hints[r])
+				t.Errorf("%v: Expected result to be %#v, got %#v", tc.description, tc.expectedHints[r], hints[r])
```

Comment:
Is there a reason for adding the `#` here? None of these types implement the `GoStringer` interface.

Reply:
With this change I was able to get:
```
topology_hints_test.go:105: Single Request, no alignment: Expected result to be []topologymanager.TopologyHint(nil), got []topologymanager.TopologyHint{}
```

without it I get:
```
topology_hints_test.go:105: Single Request, no alignment: Expected result to be [], got []
```

Reply:
I guess its just to have more verbose output include the struct fields, etc?

Reply:
that was the intention. without the modification it's impossible to say why the test failed when I added `Dev3` alone